assets/app/components/linkButton.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const propTypes = {
+  className: React.PropTypes.string,
+  text: React.PropTypes.string,
+  href: React.PropTypes.string
+}
+
+const LinkButton = ({className, text, href}) =>
```

Comment:
React will magically call createClass under the hood! It also infers that
this is a component with just a render method and no additional logic.

On Thu, Jun 23, 2016 at 8:54 AM, Jeremia Kimelman <notifications@github.com>
wrote:

> In assets/app/components/linkButton.js
> <https://github.com/18F/federalist/pull/383#discussion_r68261461>:
>
> > @@ -0,0 +1,16 @@
> > +import React from 'react';
> > +
> > +const propTypes = {
> > +  className: React.PropTypes.string,
> > +  text: React.PropTypes.string,
> > +  href: React.PropTypes.string
> > +}
> > +
> > +const LinkButton = ({className, text, href}) =>
>
> Woah, you can make components with just function like this?
>
> What does React do in this case?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/18F/federalist/pull/383/files/22583f28e136c0f830dece564a844391151a71fa..a447a0c8bd8a39070b5b1511dc66244db7102240#r68261461>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABWyGM5VJStDv0HnFVUTpQlu3LQpLj6Wks5qOqwegaJpZM4I67ik>
> .
>